methods.tex

```diff
@@ -29,7 +29,7 @@ \subsection*{Library organization}
 
 The \code{ndarray} is the data structure at the heart of NumPy.
 The data structure stores regularly strided homogeneous data types
-in a contiguous block memory, allowing for the efficient representation
+inside a contiguous block memory, allowing for the efficient representation
```

Comment:
not sure it makes a difference, I stumbled a bit because a sliced array itself is not contiguous anymore, of course it still  is a "view" into a contiguous block.

Reply:
Good point!